### DISCUSSION TITLE
withdraw asset

### DISCUSSION THREAD

**Discussion by Baghalishahi** [blockscout/blockscout#10660] [Q&A]
hi guys. I bridged my asset to worldchain with merkly and can't withdraw it. please guide me.

**Comment by Sentini14** [comment]
I hope you are okay with what you are looking for ? 😊

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
